mayastor/src/bin/mayastor-client/pool_cli.rs

```diff
@@ -205,3 +218,22 @@ fn pool_state_to_str(idx: i32) -> &'static str {
         rpc::PoolState::PoolFaulted => "faulted",
     }
 }
+
+fn pooltype_to_str(idx: i32) -> &'static str {
```

Comment:
calling pool_type_to_str(3) will crash the program the whole idea here is that you can only pass valid values to PoolType::from_i32().  So it's probably better to write something as:

`PoolType::from_i32(idx)?` and handle the error

Reply:
This function is only for printing pooltype in the mayastor-client `pool list`. So I guess the wrong index should not come.